packages/web-pkg/src/components/CreateLinkModal.vue

```diff
@@ -384,6 +404,10 @@ export default defineComponent({
   justify-content: flex-start;
 }
 
+.link-modal-password {
+  margin-left: calc(var(--oc-space-small) + 22px);
```

Comment:
I guess this is the alignment of `Password` with `Anyone can...`? Maybe better go for a "grid-based" solution if possible? So the icon in the first "column", the rest in the second so it automatically aligns. Working with pixels always feels so fragile.

Reply:
I am not sure about this, we would need to fake an empty element in order to archive this, it also makes the HTML markup less easy to read. 
Any opinions on this @kulmann 

The current implementation calculated the margin based on the icon width, which is 22px

Reply:
Why do we need to fake an empty element? What I mean is something like this:

```
<div class="oc-flex">
  <div><oc-icon ... /></div>
  <div>
    <div>role select</div>
    <div>password</div>
  </div>
</div>
```

Reply:
Because we need something like that:


icon  |   roleselect   |   options|
         |   password 

That's not gonna work with your proposal, a margin or adding an empty element to a grid in order to archive this visual appearance is inevitable

Reply:
I'm pretty sure that works with my proposal, unless I'm missing something 😅 It just needs to be extended via the third column:

```
<div class="oc-flex">
  <div><oc-icon ... /></div>
  <div>
    <div>role select</div>
    <div>password</div>
  </div>
  <div>options</div>
</div>
```

Reply:
Feel free to commit on this pr ☺️

Reply:
Hm now I think I understand what you are trying to archive 🤔 

Give me a second

Reply:
I don't know if this is the correct way to archive the desired look, as we would limit the passwords space to the right always at the same position where the options button starts

🤔🤔🤔

Reply:
Yeah that's true... hm. Well then, probably not worth investing more time into making this "beautiful". Just a nitpick anyways 🤷

Reply:
I leave this question open to my superior 😁

A second review would anyway pretty welcome

Reply:
Not worth the refactoring... would be a three-column-grid with (at least) two lines, where on line 2 the columns 2 and 3 would be merged. Most of all I see the issue that in advanced mode the role icon comes from within the role select (= one component in the modal) whereas in the simple mode it's two different elements (the icon and the span). So in advanced mode it would again be different layouting or we'd need to break the icon from the role select in advanced mode out of the role select component. Meh. I share the dislike of pixel-based design/layout, but in this case I agree that the effort is not worth it. 🙈